FFWEB-1803: Publish `dispatchRaw` function (#270)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/core-result-dispatcher.md

</script>
<script defer src="pathToFFWebComponents/dist/bundle.js"></script>
```

### `dispatchRaw(response, topics)`
___
Manually dispatches a response to the Web Components.

> Topics argument is optional - skipping it causes the response will be dispatched as standard search result.

Reviewer: Perhaps we could elaborate a bit on this.
For example why this method is useful, what its intended use case is and so on.
How does its behaviour differ when specifying `topics` compared to not specifying it?
If the response is not dispatched as a standard search result, what is it dispatched as instead?

Author: I updated the description. Hope that this level of details makes things more clear with not confusing reader in the same time

```html
<script type="text/javascript">
  document.addEventListener('WebComponentsReady', function () {
      factfinder.communication.ResultDispatcher.dispatchRaw(responseToDispatch);
  });
</script>
```

> **Note**: Make sure you put `dispatchRaw` call inside `WebComponentsReady` listener.
> That guarantees all attributes set for `ff-communication` (e.g. `version`) will be reflected as communication parameters.

Reviewer: ```suggestion
**Note**: Make sure you put the `dispatchRaw` call inside a `WebComponentsReady` listener.
This guarantees all `ff-communication` attributes (e.g. `version`) will be reflected as communication parameters.
```

</script>
<script defer src="pathToFFWebComponents/dist/bundle.js"></script>
```

### `dispatchRaw(response, topics)`
___
Manually dispatches a **response** to the Web Components.
This method is useful when there is a need to dispatch FACT-Finder response received in different way than by AJAX request, sent by Web Components.
That will be the case especially when implementing [Server Side Rendering](/documentation/3.x/server-side-rendering).

Argument **topics** is optional - skipping it causes the response will be dispatched to all default topics causing updates all the actively subscribing elements.

> With **topics** argument passed only a part of the response, related to the passed topic will be dispatched (e.g. using`suggest` will cause an update only on elements which subscribe to that topic - by default `ff-suggest-element`).

Author: Its misstype. That should be the `ff-suggest` element

> With **topics** argument passed only a part of the response, related to the passed topic will be dispatched (e.g. using`suggest` will cause an update only on elements which subscribe to that topic - by default `ff-suggest-element`).

Reviewer: I see what you mean. In this case I think the formatting in `removeCallback` is wrong. At the top under `subscribe`, monospace is used to refer to the parameters.
Allow me to adjust this quickly.

> **Note** It is possible to pass multiple topics in an array.

Reviewer: Is the note intended to be inline? Using a new paragraph like the next note might be more of an eye-catcher.
Or a forced linebreak could be enough already.
```suggestion
**Note**: It is possible to pass multiple topics in an array.
```
